crates/containerd-shim-wasm/src/sys/unix/container/executor.rs

```diff
@@ -65,6 +66,14 @@ impl<E: Engine> LibcontainerExecutor for Executor<E> {
             }
         }
     }
+
+    fn setup_envs(
```

Comment:
Can you write a comment why this needs to be a no-op?

Reply:
yeah I added a comment explaining why this is a no-ope